PurchasesTests/Purchasing/PurchaserInfoTests.swift

```diff
@@ -346,11 +350,17 @@ class BasicPurchaserInfoTests: XCTestCase {
         expect(entitlements["old_pro"]).to(beNil());
     }
 
-    func testPurchaseDate() {
+    func testPurchaseDateForEntitlement() {
         let purchaseDate = self.purchaserInfo!.purchaseDate(forEntitlement: "pro")
         expect(purchaseDate).to(equal(Date(timeIntervalSinceReferenceDate: 562288673)))
     }
 
+    func testPurchaseDateForProductIdentifier() throws {
+        let purchaserInfo = try XCTUnwrap(self.purchaserInfo)
+        let purchaseDate = try XCTUnwrap(purchaserInfo.purchaseDate(forProductIdentifier: "threemonth_freetrial"))
+        expect(purchaseDate).to(equal(Date(timeIntervalSince1970: 1526797490)))
```

Comment:
I know other tests don't do it like this, but since this is a new test:

```suggestion
        expect(purchaseDate) == Date(timeIntervalSince1970: 1526797490)
```

Reply:
👍 good call